snowcap/api/protobuf/snowcap/input/v1/input.proto

```diff
@@ -10,13 +10,18 @@ message Modifiers {
 }
 
 message KeyboardKeyRequest {
-  uint32 id = 1;
+  oneof target {
+    uint32 layer_id = 1;
+    uint32 popup_id = 2;
+  }
 }
```

Comment:
About that change:

This should not be API breaking:
- This is an internal message, only called from Layer at the moment, so only a client rebuild is needed here.
- The change itself is binary wire-safe, so there should not be any issue if an older client tries to send this message without rebuilding.

Let me know if I should change it regardless :)